en/docs/setup/migrating-to-590.md

```diff
@@ -1,22 +1,15 @@
 # Migrating to 5.9.0
 
-!!! note
-    Before you follow this section, see [Before you begin](../../setup/migrating-before-you-begin) and [Preparing for 
-    migration](../../setup/migrating-preparing-for-migration) sections to read on prerequisites.
+Before you follow this section, see
+[Before you begin](../../setup/migrating-before-you-begin) and
+[Preparing for migration](../../setup/migrating-preparing-for-migration)
+sections to read on prerequisites.
```

Comment:
Shall we change this to one of these:

"to read **the** prerequisites" or "read **up on** prerequisites" or "read **about** prerequisites"